uurshin/minotaure
# Feat#5 translation

## Description

* Update some translation keys
* Update the Readme to Markdown standards
* Make i18n global

## Test

1. `npm run build`
2. `npm run preview`
3. go to [home](http://localhost:4173/)
4. open the inspector
5. navigate through the app
* chek for console error
* check for missing translations
6. read the code
